### DISCUSSION TITLE
search engine got the wrong url - router problem

### DISCUSSION THREAD

**Discussion by nzyeehaa** [joomla/joomla-cms#38118] [General]
I have an old site (recently redeveloped to J4) 06/2022
it has been rebuilt from the ground up.- extracting database menu/cat/article/user and adding to new database(all re-indexed and rebuilt)
all search results were removed from ~giggle~ search engine 12/2021 prior to rebuild - with no results in ~giggle~ search engine at launch of new site

on re-indexing the site ~giggle~ a search engine is reporting a page `website.com/products/misc/chubb`

the problem is...
- that page was archived feb 2019 prior to rebuild (so it is not a cache problem)
- at the same time its category was changed to website.com/**sold** (again - not a cache problem - since site rebuild)
- `website.com/sold` is marked `noindex, nofollow`

and the worst problems
-  - ~giggles~ search engines page link still works and takes me to the page which **should not appear (at all)** ~website.com/products/misc/chubb~
-  - the breadcrumbs show correct `home > sold> chubb` but now there is duplicate content

so its not even search engines problem
-  the page is accessible at `/sold/chubb` correctly
-  the page is accessible at ~/products/misc/chubb~ this location does not exist and should be a 404! 
-  the page is accessible at ~/products/chubb~ this location does not exist and should be a 404! 
-  the page is accessible at ~/products/123~ (item id number only) this location does not exist and should be a 404! 
### **HOW is that even possible????**
- **what is Joomla doing to our website search rankings ????**

{id numbers omitted for brevity}

setup j4.1.4
.htaccess 
SEF urls
url rewriting
system cache on - conservative

I have run out of hair to pull out - any help appreciated

**Comment by mariantanase** [comment]
What do you mean by "giggle" ?

**Reply by simbus82** [reply]
Google ;-)
@nzyeehaa please use right "Google" keyword so people can do a good search in github.

**Reply by nzyeehaa** [reply]
@mariantanase giggle = gooogle or other search engines without advertising for them

@simbus82 thanks for an attempt at answering - but i am here for an issue with Joomla SEF routing & not 2 advertise for any particular search engine. other search engines also have their issues. So the right keyword is "search engine"
